codec-dns/src/main/java/io/netty/handler/codec/dns/DefaultDnsRecordDecoder.java

```diff
@@ -184,4 +195,17 @@ public static String decodeName(ByteBuf in) {
 
         return name.toString();
     }
+
+    public static String decodeCharString(ByteBuf in) {
+        if (in.readableBytes() == 0) {
+            return null;
+        }
+        int length = (int) in.readByte();
+        if (in.readableBytes() < length) {
```

Comment:
@norrman I think we still need it if you make the method `public static` as it may be used not only in `DefaultDnsRecordDecoder#decodeRecord(ByteBuf)`